packages/ember-routing/lib/location/api.js

```diff
@@ -102,6 +102,53 @@ import { getHash } from 'ember-routing/location/util';
   in the actual URL. This is generally used for testing purposes, and is one
   of the changes made when calling `App.setupForTesting()`.
 
+  ### Extending Implementations
+
+  Ember scans the path app/locations/* for extending the Location API.
+
+  Example:
+ 
+  app/locations/hashbang.js 
```

Comment:
We should wrap the example block with code fences to allow for syntax highlighting.